### PR TITLE
Update subscription change page with new design that includes team tier

### PR DESCRIPTION
**Pull request by joshenlim**
Screens are also responsive

Before availability of team tier:
image

After team tier is available:
image

### CODE REVIEW

File: studio/components/interfaces/Billing/PlanSelection/Plans/Plans.Constants.ts [hunk 1]
```diff
@@ -50,19 +49,17 @@ export const PRICING_META = {
...
     additional: 'Need more? Turn off your spend cap to Pay As You Grow ',
-    cta: 'Get Started',
+    scale: 'Additional fees apply for usage and storage beyond the limits above.',
   },
   [STRIPE_PRODUCT_IDS.TEAM]: {
+    new: true,
     name: 'Team',
-    href: '#',
-    from: true,
+    description: 'For collaborative teams with different permissions and access.',
```
Comment (alaister) [inline studio/components/interfaces/Billing/PlanSelection/Plans/Plans.Constants.ts:58]: Should this be "For collaborative teams with different permissions and access **patterns**"? Or something similar?
Bonus points if you can think of shorter wording, too, so these line up:
Screenshot 2023-01-12 at 11 29 59 pm

**Review by alaister — COMMENTED** [review]
Maybe just one small wording thing but otherwise looks great!

File: studio/components/interfaces/Billing/PlanSelection/Plans/Plans.Constants.ts (hunk 1 above)
Comment (joshenlim) [reply studio/components/interfaces/Billing/PlanSelection/Plans/Plans.Constants.ts:58]: ahh right! Think i'll go with `Collaborate with different permissions and access patterns`

**Review by saltcod — APPROVED** [review]
